ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -258,21 +258,25 @@ def next_record(self) -> Iterable[Entity]:
             schema_names = inspector.get_schema_names()
             for schema in schema_names:
                 # clear any previous source database state
-                self.database_source_state.clear()
-                if filter_by_schema(
-                    self.source_config.schemaFilterPattern, schema_name=schema
-                ):
-                    self.status.filter(schema, "Schema pattern not allowed")
-                    continue
+                try:
```

Comment:
Added try catch because in some connectors like Presto -> get_view_names isn't implemented which stopped the workflow